.golangci.yml

```diff
@@ -99,11 +99,11 @@ issues:
       - staticcheck
     path: (data_source_outputs|resource_tfe_agent_pool|resource_tfe_notification_configuration|resource_tfe_organization|resource_tfe_registry_module|
       -|resource_tfe_run_trigger|resource_tfe_team_access|resource_tfe_organization_membership|resource_tfe_policy_set|resource_tfe_team_member|
-      -|resource_tfe_workspace|resource_tfe_team_organization_member|resource_tfe_variable_set)\.go
+      -|resource_tfe_workspace|resource_tfe_team_organization_member|resource_tfe_variable_set|resource_tfe_team_project_access)\.go
```

Comment:
Oh huh, I didn't know about these massive piles of exemptions living in this config file. I don't really like it! 

Or rather: having global or big glob (`_test.go`) exemptions in here seems great. 👍🏼 The _per-file_ exemptions feel like sweeping stuff under the rug, and I'd prefer if we could move them into lint-disabling comments in the affected files themselves, to make the reasoning more clear and reduce action at a distance. 

That's all a side-note for this PR though! The pattern is already extremely established, so I won't block on adding a couple more. But later I'd like us to reconsider how we do all that.